src/Bridge/Doctrine/Orm/Filter/BooleanFilter.php

```diff
@@ -0,0 +1,136 @@
+    {
+        parent::__construct($managerRegistry, $properties);
+
+        $this->booleanParameterName = $booleanParameterName;
+        $this->requestStack = $requestStack;
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * Check whether a value is equal to 1, 0, true, false (case insensitive)
+     *
+     * For each property passed, if the resource does not have such property or if the order value is different from
+     * 1, 0, true, false (case insensitive) the property is ignored.
+     *
+     */
+    public function apply(QueryBuilder $queryBuilder, string $resourceClass, string $operationName = null)
+    {
+        $request = $this->requestStack->getCurrentRequest();
+        if (null === $request) {
+            return;
+        }
+
+        $properties = $this->extractProperties($request);
+
+        foreach ($properties as $property => $boolean) {
+            if (!$this->isPropertyEnabled($property) || !$this->isPropertyMapped($property, $resourceClass)) {
+                continue;
+            }
+
+            if (empty($boolean) && isset($this->properties[$property])) {
+                $boolean = $this->properties[$property];
+            }
+
+            $boolean = strtoupper($boolean);
+         if (!in_array($boolean, ['0', '1','TRUE','FALSE'])) {
+                continue;
+            }
+
+            $alias = 'o';
+            $field = $property;
+
+            if ($this->isPropertyNested($property)) {
+                $propertyParts = $this->splitPropertyParts($property);
+
+                $parentAlias = $alias;
+
+                foreach ($propertyParts['associations'] as $association) {
+                    $alias = QueryNameGenerator::generateJoinAlias($association);
+                    $queryBuilder->leftJoin(sprintf('%s.%s', $parentAlias, $association), $alias);
+                    $parentAlias = $alias;
+                }
+
+                $field = $propertyParts['field'];
+            }
+            $valueParameter = QueryNameGenerator::generateParameterName(sprintf('%s_%s', $field, $boolean));
+
+            $queryBuilder
+                ->andWhere(sprintf('%s.%s = :%s', $alias, $field, $valueParameter))
+                ->setParameter($valueParameter, $boolean);
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDescription(string $resourceClass) : array
+    {
+        $description = [];
+
+        $properties = $this->properties;
+        if (null === $properties) {
+            $properties = array_fill_keys($this->getClassMetadata($resourceClass)->getFieldNames(), null);
+        }
+
+        foreach ($properties as $property => $boolean) {
+            if (!$this->isPropertyMapped($property, $resourceClass)) {
+                continue;
+            }
+
+            $description[sprintf('%s[%s]', $this->booleanParameterName, $property)] = [
+                'property' => $property,
+                'type' => 'string',
```

Comment:
It is ok to filter only on boolean like in the DateFilter or do we have to include all type of property ?